pkg/plugin/plugin.go

```diff
@@ -266,13 +266,10 @@ func CmdDel(args *skel.CmdArgs) error {
 
 	// Delete can be called multiple times, so don't return an error if the
 	// device is already removed.
-	err = ns.WithNetNSPath(args.Netns, func(ns.NetNS) error {
-		err = ip.DelLinkByName(args.IfName)
-		if err != nil && err == ip.ErrLinkNotFound {
-			return nil
-		}
+	_, err = withNetNS(args.Netns, "ip", "link", "del", "dev", args.IfName)
+	if err != nil && strings.Contains(err.Error(), "Cannot find device") {
```

Comment:
the logic is inverse, if you have an error which is not "cannot find device" then return an error

Reply:
Need to be `if err != nil && !(strings.Contains(err.Error(), "Cannot find device"))`